File: app/src/main/java/com/neome/api/meta/base/dto/EntVdSsRemoveRows.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdSsRemoveRows extends EntVdAutoStep
{
  @Nullable
  public StudioMapOfCondition filterCondition;

  @Nullable
  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdSsRemoveRows.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStep
import com.neome.api.meta.base.Types.MetaIdSpreadsheet
import com.neome.api.meta.base.dto.StudioMapOfCondition

interface EntVdSsRemoveRows : EntVdAutoStep
{
  val filterCondition: StudioMapOfCondition?
  val spreadsheetId: MetaIdSpreadsheet?
}